cmd/csi-driver/csi-driver.go

```diff
@@ -26,15 +26,17 @@ const (
 	csiControllerLogFile = "/var/log/hpe-csi-controller.log"
 	csiNodeLogFile       = "/var/log/hpe-csi-node.log"
 	csiEndpoint          = "unix:///var/lib/kubelet/csi.hpe.com/csi.sock"
+	csiScrubberInterval  = 1800 // Default value is 1800 seconds or 30 minutes
```

Comment:
Should we make it clear that this is an inline volume scrubber?  Just calling it csiScrubberInterval can lead to confusion... what exactly are we scrubbing?

Reply:
I thought of keeping it generic so that any future scrubbing tasks can be piggy backed. However, it makes sense to keep to specific. Here we are scrubbing ephemeral pods

Reply:
I'm curious though.  Why do we feel that we need to expose this to the user via our flag variables?  Is this something we expect them to change?

Reply:
The idea is to keep it configurable so that we can update the deployment yamls to modify it accordingly to control of frequency of scrubbing. Scrubber does traverses the pods directory path recursively `/var/lib/kubelet/pods/`.

Reply:
I'm not questioning why we allow it to be configurable.  I'm just wondering why we think it needs to be user facing.  We can treat it like we do the volume limit and make it an environment variable.

Reply:
Yes, I think that would be more ideal. Let me fix it that way instead?